src/httpserver/http_resource.hpp

```diff
@@ -31,15 +31,14 @@
 #include <map>
 #include <memory>
 #include <string>
+#include <utility>
 
+#include "httpserver/http_request.hpp"
 #include "httpserver/http_response.hpp"
 
 namespace httpserver
 {
 
-class webserver;
```

Comment:
I am surprised that we can just remove this given the friend below

Reply:
See comment above.